### PR TITLE
FIX: Add WordPress URL

### PR DESCRIPTION
**Pull request by erwinromkes**

### CODE REVIEW

File: view/adminhtml/templates/cms/wordpressdata.phtml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+$viewModel = $block->getViewModel();
+
+$wordPressUrl = $viewModel->getWordPressUrl();
+?>
+<?php if ($wordPressUrl):?>
+<script>
+    require([
+        'jquery'
+    ], function($){
+        $(document).ready(function() {
```
Comment (tdgroot) [inline view/adminhtml/templates/cms/wordpressdata.phtml:13]: Why does this need to be set on document ready? And why using jquery?

File: ViewModel/WPCIData.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    /**
+     * Get the WordPress Base Url
+     *
+     * @return string|null
+     */
+    public function getWordPressUrl(): string
+    {
+        foreach ($this->remotePageRepository->getSites() as $site) {
```
Comment (tdgroot) [inline ViewModel/WPCIData.php:30]: What happens if no sites are available?

File: ViewModel/WPCIData.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    ) {
+        $this->remotePageRepository = $remotePageRepository;
+    }
+
+    /**
+     * Get the WordPress Base Url
+     *
+     * @return string|null
+     */
+    public function getWordPressUrl(): string
```
Comment (tdgroot) [inline ViewModel/WPCIData.php:28]: Should be `?string`

File: ViewModel/WPCIData.php (hunk 2 above)
Comment (tdgroot) [inline ViewModel/WPCIData.php:30]:
```suggestion
        /** @var string|null $url */
        $url = null;
        foreach ($this->remotePageRepository->getSites() as $site) {
```

File: ViewModel/WPCIData.php [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+use Mooore\WordpressIntegrationCms\Model\RemotePageRepository;
+
+class WPCIData implements ArgumentInterface
+{
+    /**
+     * @var RemotePageRepository
+     */
+    private $remotePageRepository;
+
+    public function __construct(
```
Comment (tdgroot) [inline ViewModel/WPCIData.php:17]: Multiline not needed here

Comment (tdgroot) [inline ViewModel/WPCIData.php:17]:
```
    public function __construct(RemotePageRepository $remotePageRepository)
    {
```

**Review by tdgroot — CHANGES_REQUESTED** [review]

File: view/adminhtml/templates/cms/wordpressdata.phtml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+            window.wordpressurl = '<?php echo $wordPressUrl; ?>';
```
Comment (poespas) [inline view/adminhtml/templates/cms/wordpressdata.phtml:14]: What will happen when multiple CMS-envs are set to the installation? Will it only use one to calculate all editor URLs with?

File: view/adminhtml/layout/cms_page_edit.xml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="admin-1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceContainer name="content">
+            <block name="wpci_data" before="before.body.end"
```
Comment (GrimLink) [inline view/adminhtml/layout/cms_page_edit.xml:5]: If you use multiline code make to wrap all attributes

File: view/adminhtml/layout/cms_page_edit.xml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="admin-1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
```
Comment (GrimLink) [inline view/adminhtml/layout/cms_page_edit.xml:2]: Lines larger than 120 characters should be wrapped

**Comment by GrimLink** [comment]
@erwinromkes this PR is closed since it is stale.
And not PR should merge to the main branch!
Please move all code logic back to the your branch from the latest develop branch instead.

**Pull request closed without merging**